.gitignore

```diff
@@ -9,3 +9,4 @@ node_modules/
 !package*.json
 !.cspell.json
 *.DS_Store
+values/gatekeeper/constraints/*
```

Comment:
ok, let me explain what is happening here:
The Konstraint library is used to generate these files.  
I believe we don't want to store them in git because they are cloud/cluster specific, generated each time when doing templating for a specific context.
They are volatile - i.e. will be deleted and re-created dynamically, depending on the current context.

Reply:
Is it possible to generate these files in /tmp folder ?

Reply:
Yes, changed it to `/tmp/otomi/constraints`.